odxtools/physsegment.py

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import Any
+from xml.etree import ElementTree
+
+from .element import IdentifiableElement
+from .exceptions import odxrequire
+from .odxdoccontext import OdxDocContext
+from .odxlink import OdxLinkDatabase, OdxLinkId
+from .snrefcontext import SnRefContext
+from .utils import dataclass_fields_asdict, read_hex_binary
+
+
+@dataclass(kw_only=True)
+class PhysSegment(IdentifiableElement):
+    fillbyte: int | None = None
```

Comment:
```suggestion
    fill_byte: int | None = None
```

Reply:
the XSD calls the tag `FILLBYTE`...

Reply:
it's a clear typo in the spec, not the first or last that we run into, no need to follow it

Reply:
In contrast to the incorrect pluralizations ("SECURITYS", "FLASH-CLASSS", ...) and definitive typos ("POS-RESPONSE-SUPPRESSABLE"), I'm actually not so sure in this instance: Even in English, words are sometimes amalgamated (e.g., `bytearray`). I tend to think that `DATABLOCK` should better be `DATA-BLOCK`, but even there I am not 100% sure and if in doubt we should IMO stick with what the spec uses...